doc/ImplementingApplications/ImplementingApplication/ConceptOfAuthenticationAuthorization/ConceptOfAuthenticationAuthorization.md

```diff
@@ -0,0 +1,21 @@
+## Authorization
+In all the approved applications of ApplicationPattern , 
+* OAM services are authorized by using BasicAuth 
+* Both the Individual and Basic services are authorized by using an operation-key
+
+### OAM service :
+
+ ![BasicAuth](Images/BasicAuth.png)
+* Every OAM service request is authorized by using BasicAuth. 
+* Usually, an administrator shall access the OAM services by using a username and password. 
+* When an application receives an OAM request , the BasicAuth will be validated with the help of one of the Tiny Application Controller called “AdministratorAdministration”(AA). 
```

Comment:
please change it as "with the help of one of the applications in Tiny Application Controller"

Reply:
Thank you. Changed the statement as proposed.